babel.config.js

```diff
@@ -48,9 +48,10 @@ const config = {
 		[
 			'@babel/transform-runtime',
 			{
+				corejs: false, // we polyfill so we don't need core-js
 				helpers: true,
-				polyfill: false,
 				regenerator: false,
+				useESModules: false,
```

Comment:
Why not use Babel helpers with import/export syntax?

Reply:
The `useESModules: true` indeed looks interesting: the Babel helpers (i.e., `node_modules/@babel/runtime/helpers/esm/inherits.js`) are then imported in a ES6 module rather than CommonJS version. There is negligible impact on build size though.

Reply:
Right, given that polyfills are off, it probably doesn't matter.